src/fparser/two/Fortran2008.py

```diff
@@ -334,18 +335,6 @@ def match(fstring):
         '''
         return EndStmtBase.match('SUBMODULE', Submodule_Name, fstring)
 
-    def get_name(self):  # C1114
```

Comment:
This is a re-implementation of ``EndStmtBase.get_name()`` (https://github.com/stfc/fparser/blob/master/src/fparser/two/utils.py#L1206)